### DISCUSSION TITLE
Running conda on Apple Silicon with zsh AND bash

### DISCUSSION THREAD

**Discussion by jeffreykstone** [biostars/biostar-handbook-issues#198] [Q&A]
Is there a way that we can set our path to run conda from zsh AND bash? 

I already had miniforge on my machine (2021 MacBook Pro M1 Max) and can only get it to run when I am using zsh and not when I am using bash. 

I really don't want to uninstall everything and do a re-install in bash.

I am trying to write a script so I can run conda in bash as well and thus install bioinfo. 

Has anyone else made this work?

Thank you for any suggestions!

**Comment by ialbert** [comment]
zsh is similar to bash and mostly compatible

and I would expect that everything would work fine if you kept  zsh by default and installed everything that way - perhaps the .bash_profile should be just .profile

the reason we recommend bash is because we are not using zsh and there may some minor issues that pop up and we'd like to keep the instructions identical

but if you do use zsh then probably you can work around whatever issue
